Policy support in API (#141)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/run-tasks/run-task-stages-and-results.mdx

The run task stage status is found in `data.attributes.status`, and you can reference the following list of possible values.

| Status              | Description                                                                                                         |
|-------------------- |-------------------------------------------------------------------------------------------------------------------- |
| `pending`           | The initial status of a run task stage after creation.                                                              |
| `running`           | The run task stage is executing one or more tasks, which have not yet completed.                                    |
| `passed`            | All of the run task results in the stage passed.                                                                    |
| `failed`            | One more results in the run task stage failed.                                                                      |

> | `awaiting_override` | Indicates that the task stage is waiting for user input, once overridden it re-enters back into the `running` state |

Reviewer: ```suggestion
| `awaiting_override` | The task stage is waiting for user input. Once overridden, the run returns to the `running` state. |
```

| `running`           | The run task stage is executing one or more tasks, which have not yet completed.                                    |
| `passed`            | All of the run task results in the stage passed.                                                                    |
| `failed`            | One more results in the run task stage failed.                                                                      |

> | `awaiting_override` | The task stage is waiting for user input. Once overridden, the run returns to the `running` state.                  |

Reviewer: Also just checking - there's nothing special about these for OPA specifically is there? Or...I just want to make sure that these all apply to both

```suggestion
| `awaiting_override` | The task stage is waiting for user input. Once a user manually overrides the failed policy checks, the run returns to the `running` state.                  |
```

Author: @glennsarti Correct me if I am wrong, but. I think this is applicable only to OPA as of now, but will apply to both OPA and sentinel in the future, when sentinel is also part of the task stage?

Reviewer 2: Whoops didn't see this ping. So the verbiage we're using at the moment is "It's a Policy Override". We've tried to avoid "Policy Checks" because that's the language for current Sentinel.  While the old text is _technically_ correct, I think Laura's text is more user facing.

The GET endpoints above can optionally return related resources, if requested with [the `include` query parameter](/cloud-docs/api-docs/#inclusion-of-related-resources). The following resource types are available:

| Resource             | Description                                               |
|--------------------- |---------------------------------------------------------- |
| `run`                | Information about the associated run.                     |
| `run.workspace`      | Information about the associated workspace.               |
| `task-results`       | Information about the results for a task-stage.           |

> | `policy-evaluation`  | Information about the policy evaluation for a task-stage. |

Reviewer 2: This should be plural.

```suggestion
| `policy-evaluations`  | Information about the policy evaluations for a task-stage. |
```

> | `run`                | Additional information about the associated run.                     |
> | `run.workspace`      | Additional information about the associated workspace.               |
> | `task-results`       | Additional information about the results for a task-stage.           |
> | `policy-evaluation`  | Additional information about the policy evaluation for a task-stage. |

Reviewer: ```suggestion
| `run`                | Information about the associated run.                     |
| `run.workspace`      | Information about the associated workspace.               |
| `task-results`       | Information about the results for a task-stage.           |
| `policy-evaluation`  | Information about the policy evaluation for a task-stage. |
```
